main/http_server/axe-os/src/app/services/system.service.ts

```diff
@@ -47,8 +47,8 @@ export class SystemService {
           stratumPort: 21496,
           fallbackStratumURL: "test.public-pool.io",
           fallbackStratumPort: 21497,
-          stratumUser: "bc1q99n3pu025yyu0jlywpmwzalyhm36tg5u37w20d.bitaxe-U1",
```

Comment:
This is mock data for testing and development. I think we should keep a valid btc address here.

Reply:
Yes, it seemed like it was mock data. I guess the question is, given bitaxe's growth in popularity and the corresponding expectation that most new eyes on the project will trend toward less experienced, does the convenience of hardcoding a valid address outweigh the downstream public relations effort needed to squelch misinformed accusations of foul play on twitter, etc. I thought it would be cool to make the repo so far above reproach that there isn't even a single valid address present at all, which makes a good proactive talking point for skot when discussing the project publicly, etc.